options/options_parser.cc

```diff
@@ -656,8 +660,17 @@ Status RocksDBOptionsParser::VerifyRocksDBOptionsFromFile(
     const std::vector<ColumnFamilyOptions>& cf_opts,
     const std::string& file_name, FileSystem* fs,
     OptionsSanityCheckLevel sanity_check_level, bool ignore_unknown_options) {
+  // We infer option file readhead size from log readahead size.
+  // If it is not given, use 512KB.
+  size_t file_readahead_size = db_opt.log_readahead_size;
+  if (file_readahead_size == 0) {
+    const size_t kDefaultOptionFileReadAheadSize = 512 * 1024;
```

Comment:
Move `kDefaultOptionFileReadAheadSize` to a header or other centralized location so that it can be used elsewhere?

Reply:
I think Google C++ Style asks us for declare variable closest to where it is used. If it would be used in other places, then we can move it to globally.